README.rst

```diff
@@ -242,9 +246,9 @@ All config options can be overruled by environment variables. Prefix them with
 options. It is also possible to override logging by setting "HOWITZ_LOGGING" to
 a string of a Python dict but we do not recommend it, use a config file instead.
 
-The poll interval for the events table can be changed by adding for example ``poll_interval = 30`` to
-the ``[howitz]``-section or setting the environment variable ``HOWITZ_POLL_INTERVAL`` to a new value.
-Poll interval values are in seconds and must be integers. The default value is ``60`` seconds.
+The refresh interval for the events table can be changed by adding for example ``refresh_interval = 10`` to
+the ``[howitz]``-section or setting the environment variable ``HOWITZ_REFRESH_INTERVAL`` to a new value.
+Refresh interval values are in seconds and must be integers. The default value is ``5`` seconds.
```

Comment:
Question: After the NTIE PR was merged, I hope refresh interval refers to often HTMX is set to poll the Howitz backend for NTIE updates, not to an interval of full table refreshes?